fishtank/src/cluster.ts

```diff
@@ -10,6 +10,11 @@ import { Node } from './node'
 
 export const DEFAULT_IMAGE = 'ironfish:latest'
 export const CLUSTER_LABEL = 'fishtank.cluster'
+export const DEFAULT_CONTAINER_DATADIR = '/root/.ironfish'
+
+export type NetworkDefinition = {
+  id: number
+}
```

Comment:
This should be the same NetworkDefinition from the IF SDK, it shouldn't be a custom type. Most of the properties are missing, in particular the ones for consensus parameters, which are the ones we need to customize for testing the hard fork

Reply:
See the PR description, `Temporarily use new NetworkDefinition to unblock the dependency of https://github.com/iron-fish/ironfish/pull/4302`